subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MultiProjectJvmApplicationInitIntegrationTest.groovy

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.buildinit.plugins
+
+import org.gradle.buildinit.plugins.fixtures.ScriptDslFixture
+import org.gradle.buildinit.plugins.internal.modifiers.BuildInitDsl
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.DefaultTestExecutionResult
+import org.gradle.integtests.fixtures.ToBeFixedForConfigurationCache
+import spock.lang.Unroll
+
+import static org.gradle.buildinit.plugins.internal.modifiers.Language.GROOVY
+import static org.gradle.buildinit.plugins.internal.modifiers.Language.JAVA
+import static org.gradle.buildinit.plugins.internal.modifiers.Language.KOTLIN
+import static org.gradle.buildinit.plugins.internal.modifiers.Language.SCALA
+
+class MultiProjectJvmApplicationInitIntegrationTest extends AbstractIntegrationSpec {
+    final def targetDir = testDirectory.createDir("some-thing")
+
+    def setup() {
+        executer.withRepositoryMirrors()
+        executer.beforeExecute {
+            executer.inDirectory(targetDir)
+            executer.ignoreMissingSettingsFile()
+        }
+    }
+
+    @Unroll("creates multi-project application sample for #jvmLanguage with #scriptDsl build scripts")
+    @ToBeFixedForConfigurationCache(iterationMatchers = ".*Kotlin.*")
```

Comment:
What's broken with Kotlin?

Reply:
The Kotlin plugin (from Jetbrains) is not yet CC compatible. I'll add a not to the annotation.